lib/hexpm/repository/packages.ex

```diff
@@ -66,6 +67,28 @@ defmodule Hexpm.Repository.Packages do
     update_in(package.releases, &Release.sort/1)
   end
 
+  def releases(package) do
+    package
+    |> preload()
```

Comment:
Note for myself: I could improve this and preload only the releases here, I know